cluster-logging.md

```diff
@@ -104,11 +105,25 @@ The "Storage" column shows whether the attribute is stored into a LokiStack usin
 | `service.name` | resource | journal | stream label | |
 | `systemd.t.*` | log | journal | structured metadata | |
 | `systemd.u.*` | log | journal | structured metadata | |
+| `k8s.ovn.component` | resource | OVN logs | stream label | The OVN component generating the log|
+| `k8s.ovn.sequence` | log | OVN logs | structured metadata | The sequence number indicating the log entry's order|
+| `k8s.ovn.message` | log | OVN logs | structured metadata | The main content of the log entry, can containing key-value pairs|
+| `k8s.ovn.message.*` | log | OVN logs | structured metadata | Individual key-value pairs parsed from the `ovn.message` field|
+| `k8s.auditd.event.type` | log | auditd logs | structured metadata | The type of auditd event (e.g., `SYSCALL`, `PROCTITLE`) |
```

Comment:
@alanconway thoughts on removing "event" level? Also wondering if they should not be under k8s as 'auditd' is not limited to kubernetes and is a service that is part of a linux server?

Reply:
+1 to both:
auditd.type
auditd.*

Reply:
agree